lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
       end
 
       def scalastyle_config
-        @engine_config['config'] || '/home/app/scalastyle_config.xml'
+        @engine_config['config'] || '/code/scalastyle_config.xml'
```

Comment:
Got it. So it sounds like this is analogous to a `.rubocop.yml` or `.eslintrc`. Typically, `codeclimate init` would generate one into the user's project (unless one's already present). We'll probably implement that functionality separately, so I think to de-couple the two changes, it'd be best if the engine used `/code/scalastyle_config.xml` when present and fell back to a `/home/app/scalastyle_config.xml` when not. That change can be deployed without impacting users, then we can work on `codeclimate init` after.